### DISCUSSION TITLE
HTML class naming convention

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#111] [Ideas]
styled component를 사용해서 class를 많이 사용하지 않을 것 같지만, 기억하기로 camelCase를 사용하는 분도 있고 kebab-case로 사용하는 분도 있던 것 같아 안건으로 올립니다!
- [x] kebab-case
- [ ] camelCase

**Comment by YuHyun-P** [comment]
저는 kebab-case를 사용합니당
다수결로 따르면 될 것 같네요~

**Comment by hwoo3303** [comment]
kebab-case 한표 올립니다 🙂

**Comment by Hyevvy** [comment]
저는 다수결을 따를게요 잘 안 쓸거라서 ..
